chimp.md

```diff
@@ -0,0 +1,213 @@
+# Chimp
+
+- **Team Name:** Chimp development team
+- **Payment Details:**  0x (Add later)
+
+
+## Project Overview :page_facing_up:
+
+
+### Overview
+
+Chimp Exchange is the first private decentralized exchange (DEX) offering complete anonymity for traders by leveraging end-to-end encryption (E2EE) via the Ten Protocol. With privacy becoming a critical concern in the DeFi and Web3 ecosystem, Chimp Exchange fills the gap by providing a fully encrypted trading platform, ensuring that trades, asset movements, and identities are kept private and secure. This innovative DEX prevents tracking of trades and assets, thus protecting whales, traders, and privacy-conscious users from the risks associated with public visibility on blockchain networks.
+
+Chimp Exchange aims to be the bridge to a fully private trading future, offering users the ability to trade without fear of tracking, identity disclosure, or liquidation hunting. The platform is currently live on Testnet with Cross-Chain capabilities and boasts a rapidly growing user base and community.
```

Comment:
Where can we see the source code of your currently live platform? Is it open source? Whats the network it is currently running on?